frontend/src/components/events/utils.js

```diff
@@ -0,0 +1,12 @@
+export function findPastEndIndex(events) {
+    const currentTime = Date.now();
+    let pastEnd = events.length - 1;
+    while (
+      pastEnd >= 0 &&
+      Date.parse(events[pastEnd].end.local) - currentTime > 0
```

Comment:
i think we want to look at `start.local` instead of `end.local`? i think if an event has already started we would want to show it in past events?